[2025-05-22 16:07:26,102 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:07:26,103 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:07:26,162 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:07:26,163 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:09:00,080 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:09:00,081 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:09:00,126 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:09:00,127 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:09:00,146 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:09:00,147 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:09:00,157 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:09:00,158 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:09:00,162 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:09:00,162 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:09:00,173 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:09:00,174 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:10:40,769 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:10:40,771 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:10:40,796 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:10:40,797 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:10:40,822 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:10:40,823 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:10:40,853 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:10:40,853 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:10:40,857 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:10:40,857 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:11:01,047 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:11:24,358 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:11:24,359 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:11:24,384 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:11:24,384 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:11:24,402 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:11:24,402 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:11:24,423 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:11:24,424 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:11:24,438 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:11:24,439 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:11:24,448 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:11:24,449 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:11:24,449 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:11:24,449 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:12:55,293 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:12:55,294 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:12:55,315 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:12:55,316 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:12:55,325 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:12:55,325 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:12:55,343 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:12:55,344 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:14:25,826 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:14:25,828 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:14:25,851 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:14:25,851 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:14:25,898 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:14:25,898 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:16:00,938 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:16:00,939 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:16:00,975 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:16:00,975 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:16:00,984 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:16:00,985 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:16:21,227 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:16:44,561 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:16:44,561 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:16:44,584 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:16:44,584 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:16:44,607 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:16:44,607 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:16:44,626 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:16:44,627 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:16:44,642 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:16:44,643 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:16:44,652 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:16:44,653 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:18:21,129 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:18:21,131 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:18:21,153 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:18:21,153 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:18:21,173 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:18:21,174 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:18:21,186 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:18:21,187 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:18:21,210 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:18:21,211 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:19:52,159 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:19:52,160 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:19:52,205 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:19:52,205 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:19:52,238 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:19:52,238 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:19:52,261 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:19:52,262 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:20:12,423 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:20:36,012 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:20:36,013 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:22:07,717 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:22:07,718 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:22:07,757 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:22:07,757 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:22:07,778 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:22:07,779 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:22:07,788 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:22:07,789 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:23:41,823 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:23:41,824 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:23:41,845 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:23:41,846 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:23:41,858 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:23:41,859 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:23:41,863 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:23:41,864 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:23:41,874 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:23:41,874 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:23:41,875 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:23:41,876 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:25:20,763 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:25:20,764 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:25:20,784 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:25:20,785 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:25:20,795 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:25:20,796 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:25:20,815 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:25:20,815 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:25:20,816 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:25:20,817 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:26:58,033 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:26:58,034 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:26:58,087 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:26:58,087 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:26:58,119 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:26:58,119 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:26:58,123 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:26:58,124 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:28:35,476 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:28:35,477 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:28:35,497 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:28:35,497 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:28:35,507 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:28:35,507 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:28:35,510 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:28:35,511 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:28:35,520 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:28:35,521 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:28:35,521 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:28:35,522 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:30:06,715 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:30:06,715 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:30:06,733 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:30:06,734 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:30:06,743 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:30:06,743 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:30:06,747 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:30:06,747 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:31:39,692 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:31:39,693 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:31:39,725 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:31:39,726 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:31:39,729 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:31:39,730 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:31:39,739 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:31:39,739 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:33:16,566 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:33:16,567 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:33:16,616 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:33:16,617 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:33:16,647 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:33:16,647 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:33:16,651 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:33:16,652 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:34:57,475 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:34:57,476 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:34:57,495 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:34:57,495 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:36:31,962 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:36:31,963 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:36:31,984 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:36:31,984 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:36:32,014 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:36:32,015 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:38:02,937 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:38:02,938 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:38:02,958 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:38:02,958 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:38:02,990 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:38:02,990 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:38:23,191 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:38:46,745 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:38:46,746 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:38:46,766 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:38:46,766 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:38:46,781 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:38:46,781 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:40:20,697 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:40:20,698 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:40:20,717 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:40:20,717 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:40:20,727 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:40:20,727 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:40:20,744 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:40:20,745 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:41:55,671 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:41:55,672 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:41:55,693 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:41:55,693 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:41:55,712 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:41:55,713 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:41:55,744 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:41:55,745 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:43:32,835 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:43:32,836 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:43:32,858 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 16:43:32,859 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 16:43:32,880 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:43:32,881 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:43:32,914 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:43:32,915 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:43:32,918 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:43:32,919 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:43:32,932 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:43:32,932 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:43:53,164 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:44:16,519 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:44:16,520 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:44:16,564 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:44:16,564 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:44:16,582 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:44:16,583 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:44:16,613 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:44:16,614 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:44:16,615 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:44:16,615 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:45:56,056 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:45:56,057 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:45:56,102 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:45:56,102 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:45:56,124 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 16:45:56,124 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 16:45:56,137 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:45:56,137 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:45:56,142 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:45:56,142 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:45:56,152 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:45:56,152 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:46:18,045 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:46:41,557 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:46:41,558 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:46:41,606 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:46:41,606 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:47:01,866 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:47:25,362 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:47:25,363 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:47:25,406 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:47:25,406 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:47:25,446 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:47:25,446 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:47:25,457 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:47:25,458 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:50:36,848 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:50:36,849 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:50:36,896 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:50:36,897 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:50:36,929 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:50:36,929 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:50:36,933 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:50:36,934 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:50:36,944 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:50:36,944 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:52:09,951 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:52:09,952 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:52:09,994 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:52:09,995 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:53:53,830 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:53:53,831 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:53:53,992 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:53:53,993 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:53:54,107 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:53:54,108 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:53:54,120 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:53:54,121 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:53:54,162 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:53:54,163 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:54:14,989 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:54:50,805 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:54:50,806 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:54:50,889 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:54:50,890 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:54:50,919 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:54:50,920 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:56:29,453 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:56:29,454 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:56:29,530 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 16:56:29,530 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 16:56:29,567 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:56:29,568 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:56:29,569 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:56:29,570 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:56:49,985 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 16:57:15,731 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:57:15,733 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:57:15,780 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:57:15,781 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:57:15,831 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:57:15,832 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 16:57:15,832 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 16:57:15,833 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 16:58:52,039 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 16:58:52,040 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 16:58:52,084 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 16:58:52,085 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 16:58:52,124 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 16:58:52,125 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 16:58:52,134 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 16:58:52,134 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:00:28,469 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:00:28,470 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:00:28,510 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:00:28,511 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:00:28,522 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:00:28,522 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:02:10,880 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:02:10,881 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:02:10,907 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:02:10,908 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:02:10,943 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:02:10,944 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:02:10,953 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:02:10,953 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:02:10,954 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:02:10,954 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:03:47,171 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:03:47,172 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:03:47,209 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:03:47,210 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:03:47,223 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:03:47,224 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:03:47,225 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:03:47,226 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:05:19,876 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:05:19,877 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:05:19,902 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:05:19,903 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:05:19,923 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:05:19,923 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:05:19,961 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:05:19,962 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:05:19,973 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:05:19,973 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:05:19,974 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:05:19,974 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:06:56,167 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:06:56,168 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:06:56,213 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:06:56,213 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:06:56,250 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:06:56,251 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:06:56,260 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:06:56,261 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:08:29,208 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:08:29,209 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:08:29,260 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:08:29,261 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:08:29,309 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:08:29,309 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:08:29,321 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:08:29,322 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:10:02,967 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:10:02,968 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:10:02,989 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:10:02,989 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:10:03,012 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:10:03,013 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:10:03,043 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:10:03,044 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:10:03,047 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:10:03,048 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:10:03,058 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:10:03,058 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:10:03,059 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:10:03,059 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:11:43,508 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:11:43,509 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:11:43,532 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:11:43,533 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:11:43,555 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:11:43,556 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:11:43,585 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:11:43,585 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:11:43,589 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:11:43,589 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:11:43,599 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:11:43,600 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:12:03,784 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 17:12:29,017 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:12:29,018 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:12:29,063 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:12:29,064 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:12:29,085 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:12:29,086 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:12:29,120 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:12:29,120 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:12:29,121 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:12:29,121 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:14:09,946 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:14:09,948 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:14:10,001 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:14:10,002 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:14:10,022 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:14:10,022 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:14:10,033 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:14:10,034 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:14:10,037 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:14:10,038 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:14:10,048 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:14:10,048 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:14:32,658 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 17:14:57,995 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:14:57,996 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:14:58,043 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:14:58,044 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:14:58,082 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:14:58,082 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:16:31,434 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:16:31,435 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:16:31,479 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:16:31,480 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:16:31,518 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:16:31,519 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:16:31,528 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:16:31,529 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:18:04,939 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:18:04,940 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:18:04,996 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:18:04,997 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:18:05,018 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:18:05,019 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:18:05,035 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:18:05,036 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:18:05,046 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:18:05,046 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:19:38,096 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:19:38,097 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:19:38,117 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:19:38,118 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:19:38,135 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:19:38,135 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:22:09,927 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:22:09,928 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:22:09,982 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:22:09,983 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:22:10,031 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:22:10,032 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:23:42,326 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:23:42,327 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:23:42,347 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:23:42,348 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:23:42,390 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:23:42,391 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:25:15,089 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:25:15,090 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:25:15,110 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:25:15,111 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:25:15,164 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:25:15,165 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:26:51,517 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:26:51,519 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:26:51,565 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:26:51,566 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:26:51,597 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:26:51,597 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:26:51,601 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:26:51,601 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:26:51,612 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:26:51,612 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:27:14,076 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 17:27:39,245 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:27:39,246 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:27:39,269 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:27:39,270 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:27:39,290 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:27:39,291 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:27:39,326 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:27:39,326 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:27:39,328 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:27:39,328 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:29:12,820 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:29:12,821 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:29:12,843 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:29:12,844 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:29:12,866 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:29:12,866 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:29:12,885 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:29:12,885 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:29:12,913 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:29:12,913 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:29:12,914 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:29:12,914 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:30:54,181 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:30:54,182 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:30:54,209 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:30:54,210 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:30:54,233 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:30:54,233 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:30:54,259 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:30:54,260 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:30:54,290 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:30:54,291 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:31:14,571 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 17:31:39,686 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:31:39,688 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:31:39,737 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:31:39,737 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:31:39,756 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:31:39,756 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:31:39,785 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:31:39,786 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:33:12,930 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:33:12,931 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:33:12,955 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:33:12,955 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:33:12,976 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:33:12,977 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:33:12,995 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:33:12,995 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:33:13,010 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:33:13,011 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:33:13,023 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:33:13,023 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:33:33,261 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 17:33:58,369 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:33:58,370 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:33:58,394 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:33:58,395 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:33:58,415 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:33:58,416 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:33:58,440 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:33:58,440 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:33:58,450 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:33:58,450 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:33:58,466 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:33:58,466 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:35:31,514 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:35:31,515 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:35:31,537 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:35:31,537 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:35:31,548 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:35:31,548 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:35:31,552 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:35:31,553 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:35:31,563 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:35:31,563 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:35:31,564 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:35:31,564 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:37:07,386 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:37:07,387 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:37:07,408 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:37:07,408 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:37:07,431 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:37:07,431 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:37:07,450 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:37:07,450 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:37:07,467 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:37:07,468 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:38:40,573 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:38:40,574 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:38:40,623 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:38:40,624 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:38:40,644 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:38:40,644 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:38:40,671 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:38:40,671 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:40:17,224 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:40:17,225 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:40:17,248 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:40:17,249 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:40:17,270 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:40:17,271 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:40:17,282 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:40:17,283 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:40:17,287 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:40:17,288 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:40:17,302 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:40:17,303 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:41:58,220 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:41:58,221 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:43:31,643 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:43:31,645 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:43:31,671 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:43:31,672 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:43:31,696 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:43:31,696 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:43:31,717 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:43:31,718 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:43:31,734 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:43:31,735 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:45:13,746 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:45:13,747 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:45:13,770 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:45:13,770 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:45:13,790 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:45:13,790 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:45:13,829 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:45:13,829 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:45:13,848 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:45:13,849 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:46:54,053 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:46:54,054 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:46:54,164 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:46:54,165 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:46:54,361 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:46:54,362 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:46:54,364 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:46:54,365 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:48:27,542 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:48:27,543 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:48:27,586 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:48:27,587 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:48:27,606 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:48:27,607 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:48:27,617 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:48:27,617 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:48:27,633 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:48:27,634 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:48:27,634 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:48:27,635 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:50:03,228 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:50:03,229 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:50:03,268 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:50:03,269 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:50:03,281 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:50:03,282 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:51:39,548 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:51:39,549 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:51:39,600 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:51:39,601 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:51:39,635 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:51:39,636 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:51:39,653 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:51:39,654 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:53:12,844 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:53:12,845 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:53:12,896 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:53:12,897 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:53:12,930 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:53:12,931 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:53:12,935 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:53:12,936 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:53:12,948 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:53:12,949 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:54:47,603 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:54:47,604 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:54:47,625 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:54:47,625 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:54:47,649 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:54:47,649 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:54:47,667 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 17:54:47,668 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 17:54:47,694 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:54:47,695 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:54:47,695 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:54:47,695 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 17:55:07,963 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 17:55:33,235 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:55:33,236 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:55:33,258 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:55:33,259 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:55:33,282 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:55:33,283 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:55:33,313 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:55:33,313 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:55:33,317 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:55:33,317 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:55:33,326 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:55:33,327 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:57:06,926 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:57:06,928 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:57:06,953 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:57:06,954 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:57:06,986 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:57:06,987 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:57:06,993 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:57:06,994 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:57:07,005 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:57:07,006 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:58:40,455 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 17:58:40,456 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 17:58:40,480 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 17:58:40,480 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 17:58:40,503 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 17:58:40,504 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 17:58:40,535 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 17:58:40,536 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 17:58:40,540 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 17:58:40,540 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 17:58:40,550 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 17:58:40,550 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 17:58:40,550 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 17:58:40,551 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:00:14,219 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:00:14,220 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:00:14,244 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:00:14,245 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:00:14,267 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:00:14,268 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:00:14,290 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:00:14,291 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:00:14,303 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:00:14,304 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:00:14,309 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:00:14,310 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:01:49,335 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:01:49,335 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:01:49,355 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:01:49,355 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:01:49,373 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:01:49,373 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:01:49,382 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:01:49,382 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:01:49,382 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:01:49,382 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:03:22,480 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:03:22,481 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:03:22,525 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:03:22,525 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:03:22,563 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:03:22,563 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:03:22,574 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:03:22,575 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:04:55,912 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:04:55,913 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:04:55,934 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:04:55,935 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:04:55,945 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:04:55,945 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:04:55,949 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:04:55,949 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:04:55,959 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:04:55,959 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:04:55,960 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:04:55,960 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:06:29,133 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:06:29,135 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:06:29,181 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:06:29,182 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:06:29,221 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:06:29,222 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:06:29,226 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:06:29,226 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:06:29,236 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:06:29,236 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:06:29,237 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:06:29,237 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:08:12,339 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:08:12,340 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:08:12,360 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:08:12,360 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:08:12,376 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:08:12,376 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:08:12,386 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:08:12,387 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:08:12,387 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:08:12,388 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:09:45,680 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:09:45,681 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:09:45,704 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:09:45,705 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:09:45,740 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:09:45,741 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:09:45,757 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:09:45,757 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:09:45,758 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:09:45,758 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:10:06,020 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 18:10:31,182 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:10:31,183 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:10:31,226 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:10:31,227 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:10:31,249 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:10:31,249 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:10:31,259 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:10:31,260 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:10:31,264 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:10:31,264 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:10:31,274 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:10:31,274 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:12:04,511 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:12:04,512 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:12:04,550 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:12:04,550 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:12:04,563 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:12:04,564 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:12:27,032 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 18:12:52,004 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:12:52,005 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:12:52,026 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:12:52,027 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:12:52,063 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:12:52,063 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:14:25,627 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:14:25,628 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:14:25,670 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:14:25,670 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:14:25,700 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:14:25,701 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:14:25,705 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:14:25,705 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:15:59,003 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:15:59,004 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:15:59,050 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:15:59,051 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:15:59,101 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:15:59,102 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:17:35,947 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:17:35,948 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:17:35,996 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:17:35,996 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:17:36,033 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:17:36,034 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:19:09,763 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:19:09,764 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:19:09,787 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:19:09,787 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:19:09,809 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:19:09,809 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:19:09,849 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:19:09,849 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:19:09,862 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:19:09,863 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:19:30,157 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 18:19:55,256 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:19:55,257 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:19:55,302 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:19:55,303 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:19:55,334 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:19:55,335 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:19:55,339 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:19:55,339 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:19:55,352 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:19:55,353 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:21:36,834 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:21:36,835 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:21:36,861 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:21:36,862 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:21:36,886 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:21:36,886 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:23:10,457 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:23:10,457 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:23:10,492 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:23:10,492 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:23:10,502 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:23:10,502 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:23:30,866 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 18:23:55,871 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:23:55,872 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:23:55,919 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:23:55,920 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:23:55,939 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:23:55,940 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:23:55,949 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:23:55,949 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:23:55,967 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:23:55,967 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:24:18,428 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 18:24:43,543 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:24:43,544 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:24:43,569 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:24:43,570 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:24:43,588 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:24:43,589 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:24:43,604 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:24:43,604 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:24:43,614 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:24:43,615 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:26:20,097 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:26:20,098 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:26:20,122 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:26:20,122 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:26:20,142 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:26:20,142 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:26:20,162 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:26:20,163 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:26:20,178 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:26:20,178 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:26:20,188 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:26:20,188 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:27:53,692 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:27:53,693 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:27:53,716 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:27:53,716 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:27:53,736 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:27:53,737 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:27:53,753 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:27:53,754 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:27:53,765 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:27:53,766 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:29:29,816 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:29:29,817 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:29:29,836 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:29:29,837 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:29:29,874 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:29:29,875 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:29:29,885 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:29:29,886 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:31:03,563 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:31:03,564 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:31:03,588 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:31:03,589 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:31:03,612 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:31:03,612 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:31:03,663 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:31:03,663 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:32:36,103 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:32:36,104 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:32:36,125 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:32:36,126 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:32:36,162 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:32:36,162 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:32:36,172 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:32:36,173 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:34:09,139 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:34:09,140 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:34:09,162 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '-1';" to update dbms for error: -1 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-22 18:34:09,163 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-22 18:34:09,184 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:34:09,184 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:34:09,226 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:34:09,227 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:34:09,240 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:34:09,241 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:35:44,564 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:35:44,565 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:35:44,605 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:35:44,605 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:35:44,640 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:35:44,640 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:35:44,656 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:35:44,656 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:37:26,734 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:37:26,735 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:37:26,778 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:37:26,778 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:37:26,799 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:37:26,799 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:37:26,829 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:37:26,830 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:39:08,414 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:39:08,415 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:39:08,459 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:39:08,460 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:39:08,481 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:39:08,482 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:39:08,499 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:39:08,499 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:39:08,510 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:39:08,511 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:40:41,409 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:40:41,410 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:40:41,456 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:40:41,457 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:40:41,474 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:40:41,475 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:40:41,484 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:40:41,484 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:40:41,488 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:40:41,488 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:40:41,500 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:40:41,501 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:41:01,761 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 18:41:26,875 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:41:26,876 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:41:26,923 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:41:26,924 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:41:26,946 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:41:26,947 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:41:26,978 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:41:26,979 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:43:09,159 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:43:09,160 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:43:09,210 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:43:09,211 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:43:09,235 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:43:09,235 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:43:09,246 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:43:09,247 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:43:09,251 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:43:09,252 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:43:09,266 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:43:09,266 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:43:09,267 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set bgwriter_lru_maxpages to '-1';" to update dbms for error: -1 is outside the valid range for parameter "bgwriter_lru_maxpages" (0 .. 1073741823)

[2025-05-22 18:43:09,267 INFO] [postgres.py:set_knob:130] Knob bgwriter_lru_maxpages is set to 100
[2025-05-22 18:43:29,563 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-22 18:43:54,854 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:43:54,855 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:43:54,901 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:43:54,902 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-22 18:43:54,923 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_vacuum_threshold to '-1';" to update dbms for error: -1 is outside the valid range for parameter "autovacuum_vacuum_threshold" (0 .. 2147483647)

[2025-05-22 18:43:54,924 INFO] [postgres.py:set_knob:130] Knob autovacuum_vacuum_threshold is set to 50
[2025-05-22 18:43:54,934 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_multixact_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_multixact_freeze_max_age" (10000 .. 2000000000)

[2025-05-22 18:43:54,935 INFO] [postgres.py:set_knob:130] Knob autovacuum_multixact_freeze_max_age is set to 400000000
[2025-05-22 18:43:54,939 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set checkpoint_timeout to '0';" to update dbms for error: 0 s is outside the valid range for parameter "checkpoint_timeout" (30 .. 86400)

[2025-05-22 18:43:54,939 INFO] [postgres.py:set_knob:130] Knob checkpoint_timeout is set to 5min
[2025-05-22 18:43:54,950 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-22 18:43:54,951 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-22 18:45:36,093 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_max_workers to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_max_workers" (1 .. 262143)

[2025-05-22 18:45:36,094 INFO] [postgres.py:set_knob:130] Knob autovacuum_max_workers is set to 3
[2025-05-22 18:45:36,137 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-22 18:45:36,138 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
